src/context/alert.tsx

```diff
@@ -1,10 +1,40 @@
 /* eslint-disable @typescript-eslint/no-empty-function */
-import React, { createContext, FunctionComponent, useState } from "react";
+import React, {
+  createContext,
+  FunctionComponent,
+  useState,
+  useCallback
+} from "react";
 import {
   AlertModal,
   AlertModalProps
 } from "../components/AlertModal/AlertModal";
 
+export const ERROR_MESSAGE = {
```

Comment:
Same as on the backend, will an enum be appropriate here?

Reply:
good idea! changed!